src/datasets/load.py

```diff
@@ -1887,6 +2120,7 @@ def load_dataset(
     streaming: bool = False,
     num_proc: Optional[int] = None,
     storage_options: Optional[Dict] = None,
+    trust_remote_code: bool = None,
```

Comment:
I think we should raise a `FutureWarning` to warn users that the default behavior will change in the future if they do not explicitly pass `trust_remote_code`.

Reply:
good idea !